hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OFSPath.java

```diff
@@ -163,6 +163,14 @@ public String getKeyName() {
     return keyName;
   }
 
+  /**
+   * Returns the last element of the KeyName, which can either be a file name
+   * or the leaf directory/bucket.
+   */
+  public String getFileName() {
+    return keyName.substring(keyName.lastIndexOf(OZONE_URI_DELIMITER) + 1);
```

Comment:
Could use  Paths.getFilename() instead
```suggestion
    return Paths.get(keyName).getFileName().toString();
```

Reply:
thanks @sadanand48, this is a better way to check it, my approach was even not correct in the case of a path like `/vol1/bucket1/dir1/` as it returned empty. I updated my patch accordingly and also adjusted the tests.